odoo/tools/translate.py

```diff
@@ -259,18 +264,25 @@ def process(node):
             result.tail = node.tail
             has_text = (
                 todo_has_text or nonspace(result.text) or nonspace(result.tail)
-                or any((key in TRANSLATED_ATTRS and val) for key, val in result.attrib.items())
```

Comment:
That seems like a completely unnecessary pseudo-optimisation.

Reply:
TBH, if I had made that code, I would have done the same. For the sake of clarity, not optimization. Worth changing that back?